src/Microsoft.VisualStudio.Web.CodeGenerators.Mvc/RazorPage/RazorPageGeneratorModel.cs

```diff
@@ -15,10 +15,10 @@ public class RazorPageGeneratorModel : CommonCommandLineModel
         [Option(Name = "partialView", ShortName = "partial", Description = "Generate a partial view, other layout options (-l and -udl) are ignored if this is specified")]
         public bool PartialView { get; set; }
 
-        [Option(Name ="noPageModel", ShortName ="npm", Description =" Switch to not generate a PageModel class for Empty template.")]
+        [Option(Name ="noPageModel", ShortName ="npm", Description ="Switch to not generate a PageModel class for Empty template.")]
```

Comment:
Nit: the other descriptions don't end in periods, should be consistent one way or the other.